styles/themes/charcoal.css

```diff
@@ -146,7 +146,7 @@
   font-style: normal;
   font-weight: 700;
   font-display: swap;
-  src: url(https://fonts.gstatic.com/s/ebgaramond/v25/SlGDmQSNjdsmc35JDF1K5E55YMjF_7DPuGi-DPNUAw.ttf) format('truetype');
+  src: url(https://fonts.gstatic.com/s/ebgaramond/v26/SlGDmQSNjdsmc35JDF1K5E55YMjF_7DPuGi-DPNUAw.ttf) format('truetype');
```

Comment:
this was failing without even updating less. Seems the url in our less file resolves to a new version now.

Reply:
I guess this could impact production, let me sandbox this.

Reply:
This periodically happens and it's a bit frustrating. Appreciate you fixing it.